elasticsearch.md

```diff
@@ -799,11 +799,86 @@ So that Kibana will forward the necessary headers to Elasticsearch.
 
 [Impersonation](details/impersonation.md) is supported by this rule without an extra configuration.
 
-#### `groups_or` (or `groups`)
+#### `user_belongs_to_groups`
 
-`groups_or: ["group1", "group2"]`
+The ACL block will match when the user belongs to groups matching the condition defined in the `user_belongs_to_groups` config section. The information about what users belong to what groups is defined in the `users` section, typically situated after the ACL, further down in the YAML.
```

Comment:
This should be changed too. See the comment above - seems that I wrote sth we can adapt and put here ;)

Reply:
I tried to adapt your description from the discussion above , please check